code/datums/vote.dm

```diff
@@ -32,7 +32,7 @@ var/global/list/round_voters = list() //Keeps track of the individuals voting fo
 		if (mode)
 			// No more change mode votes after the game has started.
 			// 3 is GAME_STATE_PLAYING, but that #define is undefined for some reason
-			if (mode == "gamemode" && ticker.current_state >= 2)
+			if ((mode == "gamemode" || mode == "ship selection") && ticker.current_state >= 2)
```

Comment:
This is outside of the scope of the purpose of this PR. Was this tested?

Reply:
it is the merge conflict resolution